nixos/tests/supercollider.nix

```diff
@@ -0,0 +1,21 @@
+# This test checks that scsynth runs
+
+import ./make-test-python.nix ({ pkgs, ... }: let
+
+in {
+  name = "supercollider";
+
+  machine = { pkgs, lib, ... }: {
+    environment.systemPackages = with pkgs; [
+      jack2
+      supercollider
+    ];
+  };
+
+  testScript = ''
+    machine.execute('jackd -rd dummy &>/dev/null &')
+    machine.succeed('jack_wait -w -t 60')
+    machine.execute('scsynth -t 9999 &>/dev/null &')
+    machine.wait_for_open_port(9999)
+  '';
+})
```

Comment:
I am not sure if this is worth a vm tests because they are quite expensive.

Reply:
I thin it would be good to test if plugins work, so I would expand the test with an example that uses e.g. a sc3-plugins Plugin.

Reply:
I in contrast think it is a good idea to test this. :) Better to have an integration test, esp. with the jackd dependency.

Reply:
I'm indifferent to whether this gets added or not, since it is pretty basic and doesn't really prove much. I wanted to add more to this test, but didn't have the time or knowledge to. I mostly added it to begin with to make sure I wasn't breaking anything major while working on the plugin support

Reply:
I've decided to remove this test for now and will re-add it with a more worthwhile test later in a new PR (so I'll try to make sure something gets added soon-ish @Moredread!)